PSModuleDevelopment/functions/utility/Show-PSMDSyntax.ps1

```diff
@@ -0,0 +1,206 @@
+                    $null = $sbHelp.AppendLine("<c='$colorFoundAsterisk'>*</c> = Parameter was filled")
+                    $null = $sbHelp.AppendLine("<c='$colorNotFoundAsterisk'>*</c> = Mandatory missing")
+                }
+ 
+                "ShowParameters" {
+                    foreach ($parmSet in (Get-Command $commandName).ParameterSets) {
+                        # (Get-Command $commandName).ParameterSets | ForEach-Object {
+                        $null = $sb = New-Object System.Text.StringBuilder
+                        $null = $sb.AppendLine("ParameterSet Name: <c='em'>$($parmSet.Name)</c> - Parameter List")
+                        $null = $sb.Append("<c='$colorCommandName'>$commandName </c>")
+ 
+                        $parmSetParameters = $parmSet.Parameters | Where-Object name -NotIn $commonParameters
+         
+                        foreach ($parameter in $parmSetParameters) {
+                            # $parmSetParameters | ForEach-Object {
+                            $color = "$colorNonMandatoryParam"
+         
+                            if ($parameter.IsMandatory -eq $true) { $color = "$colorMandatoryParam" }
+         
+                            $null = $sb.Append("<c='$color'>-$($parameter.Name) </c>")
+         
+                            if (-not ($parameter.ParameterType -eq [System.Management.Automation.SwitchParameter])) {
+                                $null = $sb.Append("<c='$colParmValue'>PARAMVALUE </c>")
+                            }
+                        }
+         
+                        $null = $sb.AppendLine("")
+                        Write-PSFMessage -Level Host -Message "$($sb.ToString())"
+                    }
+ 
+                    $null = $sbHelp.AppendLine("")
+                    $null = $sbHelp.AppendLine("<c='$colorCommandName'>$colorCommandName</c> = Command Name")
+                    $null = $sbHelp.AppendLine("<c='$colorMandatoryParam'>$colorMandatoryParam</c> = Mandatory Parameter")
+                    $null = $sbHelp.AppendLine("<c='$colorNonMandatoryParam'>$colorNonMandatoryParam</c> = Optional Parameter")
+                    $null = $sbHelp.AppendLine("<c='$colParmValue'>$colParmValue</c> = Parameter value")
+                }
+                Default {}
+            }
+ 
+            if ($sbParmsNotFound.Length -gt 0) {
+                Write-PSFMessage -Level Host -Message "$($sbParmsNotFound.ToString())"
+            }
+ 
+            if ($IncludeHelp) {
+                Write-PSFMessage -Level Host -Message "$($sbHelp.ToString())"
+            }
+        }
+        else {
+            Write-PSFMessage -Level Host -Message "The function was unable to get the help of the command. Make sure that the command name is valid and try again."
+            Stop-PSFFunction -Message "Stopping because command name didn't return any help."
+            return
+        }
+    }
+    else {
```

Comment:
Terminating loop conditions should not be placed in a trailing else block - move above the current `if` position, save yourself a nested braces level and make the code more readable.